cpp/perspective/src/include/perspective/last.h

```diff
@@ -0,0 +1,30 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2020, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+#pragma once
+
+
+#ifdef PSP_ENABLE_PYTHON
+
+/*
+   Due to a very silly #define in python, this file needs
+   to come after any other perspective includes, which 
+   themselves have to precede any arrow includes.
+
+   The preprocessor is fun.
+
+   https://bugs.python.org/issue24643
+*/
+
+#ifdef WIN32
+#if _MSC_VER >= 1900
```

Comment:
@wesm had to duplicate this from feather, happens when including pybind and arrow on windows python3.6. I know its a python bug, but might make sense to make a jira in case people look for it down the road